### PR TITLE
Syndication E-mail Fixes and Branding Update

### PR DESCRIPTION
**Pull request by Dala0**
## What does this change?
Fixes various fields, removes some unused fields, and updates the branding to that current in August 2021.

## How to test
Make the correct changes to the Kubernetes configuration files and start a Mainstream Media syndication job via the Pluto interface.

## How can we measure success?
The fields and branding are correct.

## Images

image

### CODE REVIEW

File: CDS/scripts/syndication_email.rb [hunk 1]
```diff
@@ -165,20 +165,29 @@
...
-    puts item
-    puts values
-    wg = values
-  end
+wg = $store.substitute_string(ENV['working_group'])
+
+colour_value = "af1674"
+colour_switch = rand 1..5
+
+if colour_switch == 1
```
Comment (fredex42) [inline CDS/scripts/syndication_email.rb:173]: would be better written:

```ruby
case rand 1..5
  when 1
    colour_value="d10a10"
  when 2
    colour_value="ea5a0b"
...
end
```

It's more readable that way

see https://stackoverflow.com/questions/948135/how-to-write-a-switch-statement-in-ruby

**Review by fredex42 — CHANGES_REQUESTED** [review]
one very small change, 👍 otherwise

File: CDS/scripts/syndication_email.rb (hunk 1 above)
Comment (Dala0) [reply CDS/scripts/syndication_email.rb]: Done.

**Review by fredex42 — APPROVED** [review]
:)

**Pull request merged**